File: tapas_borne/src/main/java/com/mycompany/bornetapashouse/DTO/Categorie_tapasDTO.java
package com.mycompany.bornetapashouse.DTO;

public class Categorie_tapasDTO {
    
    private int tapas_id;
    private int categorie_id;

    public Categorie_tapasDTO(int tapas_id, int categorie_id) {
        this.tapas_id = tapas_id;
        this.categorie_id = categorie_id;
    }

    public int getTapas_id() {
        return tapas_id;
    }

    public void setTapas_id(int tapas_id) {
        this.tapas_id = tapas_id;
    }

    public int getCategorie_id() {
        return categorie_id;
    }

    public void setCategorie_id(int categorie_id) {
        this.categorie_id = categorie_id;
    }
    
        @Override
    public String toString() {
        return "CategorieDTO{" +
                "tapas_id=" + tapas_id +
                ", categorie_id='" + categorie_id + '\'' +
                '}';
    }
}

File: tapas_webservice/src/DTO/categorie_tapasDTO.php
<?php
namespace App\DTO;
class Categorie_tapasDTO implements \JsonSerializable
{
    private $tapas_id, $categorie_id;

    public function setTapasId($newtapasid)
    {
        $this->tapas_id = $newtapasid;
    }
    public function getTapasId()
    {
        return $this->tapas_id;
    }
    public function setCategorieId($newcategorieid)
    {
        $this->categorie_id = $newcategorieid;
    }
    public function getCategorieId()
    {
        return $this->categorie_id;
    }

    public function jsonSerialize()
    {
        return array(
            'tapas_id' => $this->tapas_id,
            'categorie_id' => $this->categorie_id,
        );
    }
}
